src/bake/DevServer.zig

```diff
@@ -6506,6 +6805,371 @@ pub fn onPluginsRejected(dev: *DevServer) !void {
+            .runtime_type = .Nothing,
+            .name = .init(name),
+            .message = .init(message),
+            .stack = .fromFrames(frames.items),
+            .exception = null,
+            .remapped = false,
+            .browser_url = .init(browser_url),
+        };
+
+        const stderr = Output.errorWriterBuffered();
+        defer Output.flush();
+        switch (Output.enable_ansi_colors_stderr) {
+            inline else => |ansi_colors| ctx.dev.vm.printExternallyRemappedZigException(
+                &exception,
+                null,
+                @TypeOf(stderr),
+                stderr,
+                true,
+                ansi_colors,
+            ) catch {},
+        }
+
+        var out: ArrayListUnmanaged(u8) = .empty;
+        defer out.deinit(temp_alloc);
+
+        const w = out.writer(temp_alloc);
+
+        try w.writeInt(u32, exception.stack.frames_len, .little);
+        for (exception.stack.frames()) |frame| {
+            try w.writeInt(i32, frame.position.line.oneBased(), .little);
+            try w.writeInt(i32, frame.position.column.oneBased(), .little);
+
+            const function_name = frame.function_name.value.ZigString.slice();
+            try w.writeInt(u32, @intCast(function_name.len), .little);
+            try w.writeAll(function_name);
+
+            const src_to_write = frame.source_url.value.ZigString.slice();
+            if (bun.strings.hasPrefixComptime(src_to_write, "/")) {
+                const file = ctx.dev.relativePath(src_to_write);
+                try w.writeInt(u32, @intCast(file.len), .little);
+                try w.writeAll(file);
+            } else {
+                try w.writeInt(u32, @intCast(src_to_write.len), .little);
+                try w.writeAll(src_to_write);
+            }
+        }
+
+        if (runtime_lines) |*lines| {
+            // trim empty lines
+            var adjusted_lines: [][]const u8 = lines;
+            while (adjusted_lines.len > 0 and adjusted_lines[0].len == 0) {
+                adjusted_lines = adjusted_lines[1..];
+                region_of_interest_line -|= 1;
+                first_line_of_interest += 1;
+            }
+            while (adjusted_lines.len > 0 and adjusted_lines[adjusted_lines.len - 1].len == 0) {
+                adjusted_lines.len -= 1;
+            }
+
+            try w.writeInt(u8, @intCast(adjusted_lines.len), .little);
+            try w.writeInt(u32, @intCast(region_of_interest_line), .little);
+            try w.writeInt(u32, @intCast(first_line_of_interest + 1), .little);
+            try w.writeInt(u32, @intCast(top_frame_position.column.oneBased()), .little);
+
+            for (adjusted_lines) |line| {
+                try w.writeInt(u32, @intCast(line.len), .little);
+                try w.writeAll(line);
+            }
+        } else {
+            try w.writeInt(u8, 0, .little);
+        }
+
+        r.corked(respondCorked, .{ r, out.items });
+        ctx.finalize();
```

Comment:
This is not correct. We can't assume that the request has completed. This will not work when the computer is busy enough.

Reply:
apologize for this for the billionth time. the remaining uses of .corked are as follows:

- sending "not found" for /_bun/* routes that do not match
- sending "location.reload()" for purged js bundles / js bundles from previous processes
- sending the incremental visualizer, which just calls `end` with the visualizer